From 8d35d1874cd97b76a58c195b1814c630a1c7a1b3 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 14 Jul 2020 15:35:11 +0800
Subject: [PATCH 2/3] arm64: dts: ls1046: correct rcpm base address

Based on the document of QorIQ LS1046A Reference Manual, the rcpm
base address should be 0x1ee2000. However it's not worked for target board.
So port the address from previous version with 0x1ee2140 to correct the
current wrong address.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
index eb3169cf882c..6167b290453f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
@@ -806,9 +806,9 @@
 			status = "disabled";
 		};
 
-		rcpm: rcpm@1ee208c {
+		rcpm: rcpm@1ee2140 {
 			compatible = "fsl,ls1046a-rcpm", "fsl,qoriq-rcpm-2.1+";
-			reg = <0x0 0x1ee208c 0x0 0x4>;
+			reg = <0x0 0x1ee2140 0x0 0x4>;
 			#fsl,rcpm-wakeup-cells = <1>;
 		};
 
-- 
2.17.1

